public/electron/clients/billing/client.js

```diff
@@ -0,0 +1,47 @@
+const axios = require('axios');
+
+const accountEndpoints = require('./endpoints/account');
+
+/**
+ * @class
+ * @param {Object} config - Config for billing client.
+ * @param {string} config.baseURL - Base url.
+ * @param {number} [config.timeout] - Base timeout in ms.
+ * @param {Object} [config.headers] - Base headers.
+ */
+function BillingClient({
+  baseURL,
+  headers,
+  timeout,
+}) {
```

Comment:
creating another client could not be necessary. We can create endpoints for billing and set the correct URL to those endpoints. In that way, we don’t repeat a new client class creation

Reply:
fixed 1b83e2e4658207e0880c743e1d3a51d64249ab35